app/services/map_marker_json.rb

```diff
@@ -1,9 +1,18 @@
 module MapMarkerJson
-  def self.build(organisations)
-    Gmaps4rails.build_markers(Queries::Organisations.add_recently_updated_and_has_owner(organisations)) do |org, marker|
-      yield org, marker
-    end.select do |marker|
-      marker[:lat].present? && marker[:lng].present?
-    end.to_json
+  def self.build(orgs, include_extra_organisation_data = true)
+    orgs = organisations_with_extra_data(orgs) if include_extra_organisation_data
+    Gmaps4rails.build_markers(orgs) { |*args| yield *args }
```

Comment:
Lint/AmbiguousOperator: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication. (https://github.com/bbatsov/ruby-style-guide#parens-as-args)